bpf/pid.h

```diff
@@ -5,9 +5,18 @@
 #include "bpf_helpers.h"
 #include "bpf_core_read.h"
 
+#define MAX_CONCURRENT_PIDS 1000
+
 volatile const s32 current_pid = 0;
 volatile const s32 current_pid_ns_id = 0;
 
+struct {
```

Comment:
I made a map that has pid->namespace mapping, which is written by the userspace side. Without this, we wouldn't be able to correctly send all the events we want to userspace, unless `current_pid == 0`. Having all pids run through the socket filter will be a lot of overhead, so I think it's worth having this map. 

The code used to check one pid only, now it hits the map where it was doing == before.